codex/marketplace.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: 
+title: CODEX-MARKETPLACE
+name: Codex Storage Marketplace
+status: raw
+tags: codex
+editor: 
```

Comment:
Who in the team will be the RFC editor for this RFC? Whenever this RFC is updated, this editor will need to approve the changes.

Reply:
That will probably be @dryajov?